code/modules/jobs/job_types/roguetown/church/puritan.dm

```diff
@@ -15,7 +15,7 @@
 
 	outfit = /datum/outfit/job/roguetown/puritan
 	display_order = JDO_PURITAN
-	give_bank_account = 36
+	give_bank_account = 200
```

Comment:
Please make this obvious in PR body so people dont complain when they're coin pouch gets snatched!